src/accountingservice/main.go

```diff
@@ -27,24 +26,14 @@ import (
 	"github.com/open-telemetry/opentelemetry-demo/src/accountingservice/kafka"
 )
 
-var log *logrus.Logger
 var resource *sdkresource.Resource
 var initResourcesOnce sync.Once
 
-func init() {
-	log = logrus.New()
-	log.Level = logrus.DebugLevel
-	log.Formatter = &logrus.JSONFormatter{
-		FieldMap: logrus.FieldMap{
-			logrus.FieldKeyTime:  "timestamp",
-			logrus.FieldKeyLevel: "severity",
-			logrus.FieldKeyMsg:   "message",
-		},
-		TimestampFormat: time.RFC3339Nano,
-	}
-	log.Out = os.Stdout
+func initLogger() *slog.Logger {
+	logger := slog.New(slog.NewJSONHandler(os.Stderr, nil)).With("service", "accounting")
```

Comment:
This is the place where you would use to `otelslog` bridge instead of `slog.NewJSONHandle`.

Reply:
Noted